### ISSUE TITLE
Rubocop RSpec Defaults (Metrics/BlockLength) - Thoughts?

### ISSUE THREAD

**Issue by luke-hill** [rubocop/rubocop-rspec#988]
This in theory is a tiny PR. But first I wanted to get your take on it as contributors.

In Rubocop RSpec, every project I have ever used always adds 2 overrides, or ignores.
```
Metrics/BlockLength:
  Exclude:
    - *.gemspec # For whatever the name of your gem may be
    - spec/**/_spec.rb # nearly all spec files go over 25 lines
```

Should we add new upper limits / ignores for these 2 specific files. I personally think we should and it would save me overriding these in every project I work in.

**Comment by pirj** [comment]
Sure, I agree. Also CanCan abilities. And factories especially if nested `factory`s are used.

Can we override `BlockLength` in our config without breaking it for the rest of the directories?

I'm not sure we should add gemspec inclusion here, but pretty sure you didn't mean it, just added it as another example.

**Comment by luke-hill** [comment]
I actually did sorry. Most gemspecs are over 25 lines long. When you factor in all the options that are configured.

For each example of me creating a gem I have this override in. Also to my knowledge having an override which doesn't exist doesn't crash the system.

**Comment by pirj** [comment]
```yaml
Metrics/BlockLength:
  Exclude:
    - *.gemspec
```
this is better be discussed in https://github.com/rubocop-hq/rubocop/issues then.

Otherwise, a pull request is appreciated.
So, the scenario is:
1. A repo locally excludes `BlockLength` for `app/abilities`
2. `rubocop` gem excludes `*.gemspec` in its `config/default.yml`
3. `rubocop-rspec` excludes `spec/**/_spec.rb` in its `config/default.yml`
it all works together and excludes all of that.

Chances are this may work by default, and in this case, it's enough to send two pulls, one -0 +1 to `rubocop`, another -0 +4 here, and fix it once and for all.
If it doesn't - I encourage you to look at [`inherit_mode`](https://github.com/rubocop-hq/rubocop/blob/5bcf201535d36a417c703555866c33aedf6eadb9/docs/modules/ROOT/pages/configuration.adoc#merging-arrays-using-inherit_mode).

**Comment by luke-hill** [comment]
I missed this response. Sorry. I'll actually look into doing this soon if you think it's a valid inclusion.

I agree the gemspec one should go to rubocop proper. And the rubocop-rspec should have exclude for Metrics/BlockLength.

Sorry for missing this all this time.

**Comment by luke-hill** [comment]
The gemspec one was already done. (Apologies for it being 3 months). And for some reason I wasn't picking it up. But evidently this is fixed. I'll look into the rspec one soon

**Comment by pirj** [comment]
Fixed in #1256 (unless I'm missing something).
